porting/debug-build/faq.rst

```diff
@@ -0,0 +1,131 @@
+
+Porting FAQ
+===========
+
+This is a list of common error messages and corresponding fixes people run into when porting new devices to Halium. 
+Contributions welcome!
+
+Getting ``kgsl kgsl-3d0: |_load_firmware| request_firmware(a330_pm4.fw) failed: -2 … kgsl kgsl-3d0: |adreno_init| Reading pm4 microcode failed a330_pm4.fw``
+------------------------------------------------------------------------------------------------------------------------------------------------------------
```

Comment:
These long headings don't work very well. Consider putting the full error in the text rather than the heading.

![image](https://user-images.githubusercontent.com/21966173/36597234-1c24aa2c-186e-11e8-885f-4fa6cbc90e26.png)